### PR TITLE
Universal property of contractible types w.r.t. pointed types and maps

### PR DESCRIPTION
**Pull request by tomdjong**

### CODE REVIEW

**Comment by tomdjong** [comment]
The 1000th PR! :partying_face: 
(I hope you weren't saving this for something special :sweat_smile:)

File: src/structured-types/pointed-universal-property-contractible-types.lagda.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    is-contr-equiv
+      ( Σ X (λ y → y ＝ x))
+      ( equivalence-reasoning
+          (A , a) →∗ (X , x)
+          ≃ Σ (A → X) (λ f → f a ＝ x)
+            by id-equiv
+          ≃ Σ X (λ y → y ＝ x)
+            by equiv-Σ-equiv-base
+              ( _＝ x)
+              ( equiv-universal-property-contr a c X))
```
Comment (EgbertRijke) [inline src/structured-types/pointed-universal-property-contractible-types.lagda.md:71]: I think this should just work:

```suggestion
      ( equiv-Σ-equiv-base _ (equiv-universal-property-contr a c X))
```

And then I would put this equivalence reasoning in an informal explanation of the proof. We're starting to do a lot more informal explanations of our code now.

Comment (tomdjong) [reply src/structured-types/pointed-universal-property-contractible-types.lagda.md]: Thanks! Done.

**Comment by EgbertRijke** [comment]
One more thing: in the file `foundation.universal-property-contractible-types.lagda.md` could you add a link to your new file in the `## See also` section, so that people will be able to find it?

**Comment by tomdjong** [comment]
Good suggestion. Done.

**Comment by EgbertRijke** [comment]
Enabling auto-merge 🚀

**Pull request merged**